packages-next/keystone/src/scripts/build/build.ts

```diff
@@ -6,6 +9,37 @@ import { saveSchemaAndTypes } from '../../lib/saveSchemaAndTypes';
 import { CONFIG_PATH } from '../utils';
 import type { StaticPaths } from '..';
 
+// FIXME: Duplicated from admin-ui package. Need to decide on a common home.
+export function serializePathForImport(path: string) {
+  // JSON.stringify is important here because it will escape windows style paths(and any thing else that might potentionally be in there)
+  return JSON.stringify(
+    path
+      // Next is unhappy about imports that include .ts/tsx in them because TypeScript is unhappy with them becasue when doing a TypeScript compilation with tsc, the imports won't be written so they would be wrong there
+      .replace(/\.tsx?$/, '')
+  );
+}
+
+// FIXME: Duplicated from admin-ui package. Need to decide on a common home.
+export const formatSource = (src: string, parser: 'babel' | 'babel-ts' = 'babel') =>
+  prettier.format(src, { parser, trailingComma: 'es5', singleQuote: true });
+
+const generateKeystoneConfig = async (projectAdminPath: string) => {
```

Comment:
```suggestion
const generateKeystoneConfigReexport = async (projectAdminPath: string) => {
```

v much a nit, I just want it to be clear to readers that it's not generating a keystone config, it's re-exporting the user's config